apps/client/app/offers/page.tsx

```diff
@@ -28,13 +29,14 @@ const Offers = async () => {
                   name={o.name}
                   price={o.price}
                   createdAt={o.createdAt}
+                  isPromoted={o.isPromoted}
```

Comment:
Tu jak przekazujesz wszystkie properties tego obiektu do komponentu Offer to możesz zrobić po prostu <Offer {...o} />